kvconfig.yml

```diff
@@ -0,0 +1,9 @@
+routes:
+  worker-timed-out:
+    matchers:
+      title: [ "worker-timed-out" ]
+    output:
+      type: "alerts"
+      series: "gearcmd.worker_timed_out"
```

Comment:
💯  great that we can now set up per worker alerts!
Should we also unconditionally slack timeouts to some channel? #gearman-timeouts ?